javagen/src/main/java/com/azure/autorest/model/clientmodel/ClientBuilderTrait.java

```diff
@@ -5,12 +5,9 @@
 
 import com.azure.autorest.extension.base.plugin.JavaSettings;
 import com.azure.autorest.model.javamodel.JavaBlock;
-import com.azure.core.client.traits.AzureKeyCredentialTrait;
-import com.azure.core.client.traits.ConfigurationTrait;
-import com.azure.core.client.traits.EndpointTrait;
-import com.azure.core.client.traits.HttpTrait;
-import com.azure.core.client.traits.TokenCredentialTrait;
+import com.azure.core.client.traits.*;
```

Comment:
Do we want to avoid import *?

Reply:
Yes, I will fix this. We should have a checkstyle rule for this.